docs/sources/alerting/set-up/configure-alertmanager/index.md

```diff
@@ -34,17 +34,32 @@ Grafana Alerting is based on the architecture of the Prometheus alerting system.
 
 {{< figure src="/media/docs/alerting/alerting-alertmanager-architecture.png" max-width="750px" alt="A diagram with the alert generator and alert manager architecture" >}}
 
-**Grafana Alertmanager**
+This architecture decouples alert rule evaluation from notification handling, allowing Grafana alerts to be forwarded to other alertmanagers.
 
-Grafana has its own built-in Alertmanager, referred to as "Grafana" in the user interface. It is the default Alertmanager and can only handle Grafana-managed alerts.
+Grafana can be configured to forward alerts to different types of alertmanagers. It’s important to note that each alertmanager manages its own independent alerting resources, such as:
```

Comment:
```suggestion
Grafana can be configured to forward firing alert instances to different types of Alertmanagers. It’s important to note that each Alertmanager manages its own independent alerting resources. 
```

Reply:
can we give an example here? something alsong the lines of for example, when creating an alert rule, you have to make sure that the same alertmanager is selected across all the resources that belong to that alert rule?

Reply:
https://github.com/grafana/grafana/pull/93712/commits/6a83a779f515bed75c6b720d62a69527fb4e208b has rewritten a bit this paragraph -> `Grafana can use different Alertmanagers.` 

The section "manage Alertmanager configurations" is the one that refer about routing alert instances to other Alertmanagers.  However, this is a more advanced case, and I believe this falls within the scope of a different project.

Reply:
Maybe something like Grafana can be configured to forward firing alert instances to other Alertmanagers that you can configure. It;s important to note that xxx  (see add an Alertmanager). -> in this "advanced" topic we can then provide an example of how this works. because i don't think it's clear or wasn't at least to the customer that you then had to make sure that the right alertmanager was selected across all alerting resources.

Reply:
@brendamuir two or three discussions here:

> can be configured to forward firing alert instances to other Alertmanagers that you can configure

1 - I thought again about this and decided for `Grafana can use different Alertmanagers` because `forward firing alert instances` is a particular case. But you can use Alertmanagers for other cases, for example, creating data source-managed alerts.

> because i don't think it's clear or wasn't at least to the customer

2 - the underlying issue was that Alertmanagers are independent each other. Users have to select the Alertmanager and create notification policies, contact points, templates, etc for that particular Alertmanager. 

This PR attempts to clarify that in the intro, and place the screenshot of the UI next to this info. 

3 - The case of "forward firing alert instances to external alerts" is different than creating data source-managed alerts. I just took a quick look but it seems the Alertmanager configuration JSON model should be edited, and we should then document the relevant JSON properties.